candies/cli/parsers/standard.py

```diff
@@ -78,6 +78,38 @@ def format_help(self):
     return Help
 
 
+def prefixes(signature: inspect.Signature) -> str:
+    """Constructs a prefixes string for a `signature`.
+
+    For example, if `signature` has `Opt(prefix='/')` and `Opt(prefix='+')`
+    arguments, then the resulting string would be `/+`.
+    """
+
+    def prefix_of(x: inspect.Parameter) -> str:
+        prefix = getattr(x.annotation, 'prefix', '-')
+
+        if prefix == '':
+            raise ValueError(f"Argument '{x.name}' has an invalid prefix "
+                             f"'{prefix}': "
+                             f"it is an empty string")
+
+        if len(prefix) > 1:
+            raise ValueError(f"Argument '{x.name}' has an invalid prefix "
+                             f"'{prefix}': "
+                             f"it consists of more than one character")
+
+        return prefix
+
+    params = signature.parameters.values()
+    prefixes = map(prefix_of, params)
+    prefixes = set(prefixes)
+
+    if len(prefixes) == 0:
+        return '-'
+
+    return ''.join(prefixes)
```

Comment:
What do you think about `return ''.join(prefixes) or '-'`?

Reply:
Great!